version.txt

```diff
@@ -1 +1 @@
-17.0.12.0.13+7-GA
\ No newline at end of file
+17.0.13.0.14+9-GA
```

Comment:
diff --git a/version.txt b/version.txt
index 2278dd14794..2f91f298871 100644
--- a/version.txt
+++ b/version.txt
@@ -1 +1 @@
-17.0.13.0.14+9-GA
\ No newline at end of file
+17.0.12.0.13+7-GA
\ No newline at end of file